### PR TITLE
Feature/badge

### PR DESCRIPTION
**Pull request by blacksam07**
## Summary
- Add the badge to image cover

## Trello card

[29-detalle-de-libro-iv-badge](https://trello.com/c/ZeTmAeCK/29-detalle-de-libro-iv-badge)

## Screenshots
image

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -49,4 +50,12 @@
...
   border-color: $earls-green;
   padding-bottom: 20px;
-} 
+}
+
+.badge {
+  height: 118.57px;
+  position: absolute;
+  right: -20px;
+  top: -15px;
```
Comment (betoobernal) [inline scss/detail.scss]: use the left and bottom properties for omitting the negative values.

File: index.html [hunk 2]
```diff
@@ -9,7 +9,10 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body class="container">
-  <img class="container-item book-cover" src="resources/book-cover.png" alt="book_cover">
+  <div class="container-item">
+    <img class="book-cover" src="resources/book-cover.png" alt="book_cover">
+    <img class="badge" src="resources/badge.png" alt="badge">
```
Comment (CrisVega08) [inline index.html]: I suggest use a pseudo element for this

File: index.html [hunk 3]
```diff
@@ -9,7 +9,10 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body class="container">
-  <img class="container-item book-cover" src="resources/book-cover.png" alt="book_cover">
+  <div class="container-item">
+    <img class="book-cover" src="resources/book-cover.png" alt="book_cover">
```
Comment (CrisVega08) [inline index.html]: ### 
```suggestion
    <img class="book-cover" src="resources/book-cover.png" alt="book_cover" />
```

File: index.html (hunk 2 above)
Comment (CrisVega08) [inline index.html]:
```suggestion
    <img class="badge" src="resources/badge.png" alt="badge" />
```

File: scss/detail.scss [hunk 4]
```diff
@@ -49,4 +50,12 @@
   border-bottom: solid;
   border-color: $earls-green;
   padding-bottom: 20px;
-} 
+}
+
+.badge {
+  height: 118.57px;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  height: 120px;
```

File: scss/detail.scss (hunk 1 above)
Comment (CrisVega08) [inline scss/detail.scss]: 😢

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (blacksam07) [reply index.html]: this tag don’t need this 
look :eyes: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

Comment (CrisVega08) [reply index.html]: But in wolox is mandatory, Frontend Gabo words.

File: scss/detail.scss [hunk 5]
```diff
@@ -48,4 +48,21 @@
...
   padding-bottom: 20px;
-} 
+}
+
+.badge {
+  position: relative;
+}
+
+.badge::after {
+  background-image: url('resources/badge.png');
```
Comment (CrisVega08) [inline scss/detail.scss]: use only shorthand background

File: scss/detail.scss [hunk 6]
```diff
@@ -48,4 +48,21 @@
...
+  background-size: contain;
```
Comment (CrisVega08) [inline scss/detail.scss]: Same

File: scss/detail.scss [hunk 7]
```diff
@@ -48,4 +48,21 @@
...
+  background-repeat: no-repeat;
```
Comment (CrisVega08) [inline scss/detail.scss]: Same

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: index.html [hunk 8]
```diff
@@ -9,7 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body class="container">
-  <img class="container-item book-cover" src="resources/book-cover.png" alt="book_cover">
+  <div class="container-item badge">
+    <img class="book-cover" src="resources/book-cover.png" alt="book_cover">
+  </div>
```
Comment (betoobernal) [inline index.html]: maybe this does the same.
```suggestion
    <img class="book-cover container-item badge" src="resources/book-cover.png" alt="book_cover">
```

Comment (betoobernal) [reply index.html]: the position relative passes up a level, namely, to the class container.

File: scss/detail.scss [hunk 9]
```diff
@@ -48,4 +48,19 @@
...
   border-bottom: solid $earls-green;
   padding-bottom: 20px;
-} 
+}
+
+.badge {
+  position: relative;
+}
+
+.badge::after {
```
Comment (betoobernal) [inline scss/detail.scss]: take advantage of sass for handling pseudoelements.
```suggestion
.badge {
  position: relative;
  
  &::after {}
}
```

**Review by betoobernal — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**